packages/core/src/payment/strategies/stripe-upe/stripe-upe-script-loader.ts

```diff
@@ -2,27 +2,56 @@ import { ScriptLoader } from '@bigcommerce/script-loader';
 
 import { PaymentMethodClientUnavailableError } from '../../errors';
 
-import { StripeHostWindow, StripeUPEClient } from './stripe-upe';
+import { StripeElements, StripeElementsOptions, StripeHostWindow, StripeUPEClient } from './stripe-upe';
 
 export default class StripeUPEScriptLoader {
     constructor(
         private _scriptLoader: ScriptLoader,
         private _window: StripeHostWindow = window
     ) {}
 
-    async load(stripePublishableKey: string, stripeAccount: string, locale?: string): Promise<StripeUPEClient> {
+    async getStripeClient(stripePublishableKey: string, stripeAccount: string, locale?: string): Promise<StripeUPEClient> {
+        let stripeClient = this._window.bcStripeClient;
+
+        if (!stripeClient) {
+            const stripe = await this.load();
+
+            stripeClient = stripe(stripePublishableKey, {
+                stripeAccount,
+                locale,
+                betas: ['payment_element_beta_2', 'alipay_pm_beta_1', 'link_default_integration_beta_1'],
```

Comment:
Should we move this in some kind of constant?

Reply:
@animesh1987 I didn't use it because the window is validated and replaced if It's empty, if you want, I could try to change it to constant but I think would make the approach a little more complex to read